docs/howto/operator/kubernetes.rst

```diff
@@ -53,13 +53,10 @@ Ultimately, it allows Airflow to act a job orchestrator - no matter the language
 
 How to use cluster ConfigMaps, Secrets, and Volumes with Pod?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Kubernetes cluster resources such as ConfigMaps, Secrets, and Volumes can be used with a Pod to be launched.
+Kubernetes cluster resources such as Secrets can be used with a Pod to be launched.
 Utilize the Airflow Kubernetes model classes such as:
```

Comment:
This sentence needs to be corrected. The plural is used when one class is described.

Reply:
Do we still support standard python dicts? (Line 60)

Also: why did this bit need to change -- should `Volume`  or `VolumeMount` not be used anymore?

Reply:
Fixed. @ashb yes `Volume` and `VolumeMount` would now be `k8s.V1Volume` and `k8s.V1VolumeMount`, I go into more detail on this in the UPDATING.md